docs/src/less/dox-chrome.less

```diff
@@ -213,6 +219,12 @@
 						top: @ts-unit-triple;
 					}
 				}
+				> .ts-menu > li > a {
+					padding-left: (@ts-unit / 2) * 5;
```

Comment:
we have `@ts-unit-half`, but why not write `@ts-unit * 2.5` ?

Reply:
Fixed.